inc/Config/QueryRunner/QueryRunner.php

```diff
@@ -204,13 +208,12 @@ public function execute( HttpQueryInterface $query, array $input_variables ): ar
 
 		// Set default input variables.
 		foreach ( $input_schema as $key => $schema ) {
-			if ( ! array_key_exists( $key, $input_variables ) && isset( $schema['default_value'] ) ) {
-				$input_variables[ $key ] = $schema['default_value'];
-			}
-
-			// If the input variable is required and not provided, return an error.
-			if ( ! array_key_exists( $key, $input_variables ) && isset( $schema['required'] ) && $schema['required'] ) {
-				return new WP_Error( 'remote-data-blocks-missing-required-input-variable', sprintf( 'Missing required input variable: %s', $key ) );
+			if ( ! array_key_exists( $key, $input_variables ) ) {
```

Comment:
This is meant to simplify this if check.

Reply:
The comment was deleted here, can you restore / improve it?